### PR TITLE
HPCC-18530 Confusion over path in despray

### PR DESCRIPTION
**Pull request by miguelvazq**
Display path information to not confuse the user that they have to enter anything but sub path information.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

**Comment by miguelvazq** [comment]
Documentation JIRA HPCC-18659

**Comment by GordonSmith** [comment]
Looks fine

File: esp/src/eclwatch/templates/DFUQueryWidget.html [hunk 1]
```diff
@@ -119,13 +119,14 @@
...
                                 <div data-dojo-type="dijit.Fieldset">
                                     <legend>${i18n.Target}</legend>
                                     <div data-dojo-type="hpcc.TableContainer">
                                         <input id="${id}DesprayTargetSelect" title="${i18n.DropZone}:" name="destGroup" style="width: 100%;" data-dojo-type="TargetSelectWidget" />
                                         <input id="${id}DesprayTargetIPAddress" title="${i18n.IPAddress}:" name="destIP" style="width: 100%;" required="true" data-dojo-props="trim: true" data-dojo-type="TargetSelectWidget" />
-                                        <input id="${id}DesprayTargetPath" title="${i18n.Path}:" name="destPath" required="false" style="width: 100%;" data-dojo-props="trim: true, readonly: false" data-dojo-type="TargetComboBoxWidget" />
+                                        <input id="${id}DesprayRelativePath" title="${i18n.TargetPath}:" style="width: 100%;" data-dojo-props="readonly: true, disabled:true" data-dojo-type="dijit.form.SimpleTextarea"/>
```
Comment (richardkchapman) [inline esp/src/eclwatch/templates/DFUQueryWidget.html]: Are new translations required?

Comment (miguelvazq) [reply esp/src/eclwatch/templates/DFUQueryWidget.html]: I added in hpcc.js line#639

Comment (richardkchapman) [reply esp/src/eclwatch/templates/DFUQueryWidget.html]: Sure - but we need other languages too?

Comment (miguelvazq) [reply esp/src/eclwatch/templates/DFUQueryWidget.html]: Then as suggested perhaps a 7.0.0 fix.

**Pull request closed without merging**